### PR TITLE
Subscription stream InventoryChanged update 

### PR DESCRIPTION
**Pull request by NCferrieri**
Edited files
- att-client\dist\types\ServerConnection\SubscriptionEventMessage.d.ts
- att-client\dist\types\ServerConnection\SubscriptionEvent.d.ts

### CODE REVIEW

File: src/ServerConnection/SubscriptionEvent.ts [hunk 1]
```diff
@@ -1,18 +1 @@
...
-  | 'PlayerMovedChunk'
-  | 'PlayerStateChanged'
-  | 'PopulationModified'
-  | 'ProfilingData'
-  | 'TraceLog'
-  | 'TradeDeckUsed'
-  | 'TrialFinished'
-  | 'TrialStarted'
-  | 'WarnLog';
+export type SubscriptionEvent = 'DebugLog' | 'ErrorLog' | 'FatalLog' | 'InfoLog' | 'InventoryChanged' | 'ObjectKilled' | 'PlayerJoined' | 'PlayerKilled' | 'PlayerLeft' | 'PlayerMovedChunk' | 'PlayerStateChanged' | 'PopulationModified' | 'ProfilingData' | 'TraceLog' | 'TradeDeckUsed' | 'TrialFinished' | 'TrialStarted' | 'WarnLog';
```
Comment (PolyphonyRequiem) [inline src/ServerConnection/SubscriptionEvent.ts:1]: well this isn't going to fly, switch the style back.

File: src/ServerConnection/SubscriptionEventMessage.ts [hunk 2]
```diff
@@ -1,179 +1,159 @@
 import type { CommonMessage } from './CommonMessage.js';
 import type { SubscriptionEvent } from './SubscriptionEvent.js';
-
```
Comment (PolyphonyRequiem) [inline src/ServerConnection/SubscriptionEventMessage.ts:3]: nor this, not the style

File: src/ServerConnection/SubscriptionEventMessage.ts [hunk 3]
```diff
@@ -1,179 +1,159 @@
...
 type DebugLogSubscriptionEventMessage = CommonMessage<'Subscription'> & {
-  eventType: 'DebugLog';
-  data: unknown;
+    eventType: 'DebugLog';
```
Comment (PolyphonyRequiem) [inline src/ServerConnection/SubscriptionEventMessage.ts:4]: why change indentation?  probably a different IDE setting.

File: src/ServerConnection/SubscriptionEventMessage.ts [hunk 4]
```diff
@@ -1,179 +1,159 @@
...
+        user: {
+           id: number,
+           username: string
+        },
+        itemname: string,
+        quantity: number,
+        itemhash: string,
+        material?: string,
+        savestring: string,
+        changetype: string,
```
Comment (PolyphonyRequiem) [inline src/ServerConnection/SubscriptionEventMessage.ts:31]: use the union model or whatever TS calls it, like below.

source: 'Unknown' | 'Impact' | 'FallDamage';

**Comment by mdingena** [comment]
@NCferrieri Can you please run `npm installl` followed by `npm run prepare` to ensure you have Husky installed? It enforces a pre-commit hook that will make sure the code you will be committing gets formatted like everything else. It's how I configured my code quality and style and I won't accept pull requests that use different code styles, such as different indentation, line lengths, mixed quote usage, semi-colon usage, etc.

If you install Husky you don't need to even think about it. Just commit your code and it will format your code before committing.

**Review by mdingena — CHANGES_REQUESTED** [review]
It's difficult to tell what you really changed because your IDE changed lots of unrelated things such as indentation. Please enforce the project's code style using ESLint and Prettier.

**Comment by TheRavenSeb** [comment]
Understood we'll fix that together @NCferrieri

**Review by mdingena — APPROVED** [review]
Thanks for your contribution. :)

**Pull request merged**